xknx/core/telegram_queue.py

```diff
@@ -140,7 +145,7 @@ async def process_telegram_incoming(self, telegram):
                 ret = await telegram_received_cb.callback(telegram)
                 if ret:
                     processed = True
-
+        # TODO: why don't we process every incoming telegram by device?
```

Comment:
The callbacks usually come from value_reader. I think if a knx-device responds, we should update all fo our devices that use the GA. 
Another source for these callbacks would be HAs fire_event where we use 
```python
        # False signals XKNX to proceed with processing telegrams.
        return False
```
so `processed` is False.